modules/update-upgrading-cli.adoc

```diff
@@ -37,6 +37,13 @@ NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
 version   4.9.23     True        False         158m    Cluster version is 4.9.23
 ----
 
+. Set your channel to `stable-4.10`:
```

Comment:
probably worth linking from here to [the channel docs][1], because `fast-4.10` or `eus-4.10` would be supported choices as well.

[1]: https://docs.openshift.com/container-platform/4.10/updating/understanding-upgrade-channels-release.html

Reply:
@wking Done. I have pushed the changes. Please check and let me know if it makes sense.